things/api/things/http/transport.go

```diff
@@ -105,6 +105,13 @@ func MakeHandler(tracer opentracing.Tracer, svc things.Service) http.Handler {
 		opts...,
 	))
 
+	r.Post("/things/list", kithttp.NewServer(
```

Comment:
Can we add a search here `/things/search`? Like search is a resource itself. Then  `POST` makes sense because you are creating a new search resource with a req body (containing search criteria).